pkg/features/kube_features.go

```diff
@@ -67,6 +67,14 @@ const (
 	// beta: v1.30
 	AppArmorFields featuregate.Feature = "AppArmorFields"
 
+	// owner: @liggitt
+	// kep:
+	// alpha: v1.31
+	//
+	// Make the Node authorizer use fine-grained selector authorization.
+	// Requires AuthorizeWithSelectors to be enabled.
+	AuthorizeNodeWithSelectors featuregate.Feature = "AuthorizeNodeWithSelectors"
```

Comment:
Huh.  I'm surprised by the addition.  Concerned about clients using kubelet creds to read pods in general?

Reply:
Yeah... we know people are piggy backing on kubelet creds, so it seems possible. I expect to graduate them together, but this tightening seemed logically separate from the "send more info to webhooks" bit and more likely to need a one-release escape hatch to me.